Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -496,5 +496,67 @@ protected override void ProcessRecord()
         }
     }
 
-    
+    /// <summary>
+    /// <para type="synopsis">
+    /// Requests that a running BQ Job be canceled.
+    /// </para>
+    /// <para type="description">
+    /// Requests that a job be canceled. This call will return immediately, and the client 
+    /// will need to poll for job status to see if the cancel completed successfully. 
+    /// Canceled jobs may still incur costs. This cmdlet returns a Job object if successful.
```

Comment:
Yep.  If the server responds with null or if any other exception is thrown by the api, the error will bubble up to the terminal.

Is this something I should explicitly mention?  I figured it was something that would be obvious to the user, but if not, I'll just add that to my list of fixes for all cmdlets.